ports/ntuples/portfile.cmake

```
@@ -0,0 +1,46 @@
+
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO RPeschke/ntuples
+    REF v0.1.2
+
+    SHA512 0cd390e79640f0b03b5ac7b58ab52996a0a69f13cd52b404acf06601c2cf5788fe6ef8c082c35ed734fa4094a0b9543a6c07a1f6deaffd8a69475abbae7268dc
+)
+
+# 1) Install headers (copy contents of core/include into <pkg>/include)
+file(INSTALL "${SOURCE_PATH}/core/include/" DESTINATION "${CURRENT_PACKAGES_DIR}/include")
+
+# 2) Install a relocatable CMake config so find_package(ntuples CONFIG) works
+file(MAKE_DIRECTORY "${CURRENT_PACKAGES_DIR}/share/ntuples")
+file(WRITE "${CURRENT_PACKAGES_DIR}/share/ntuples/ntuples-config.cmake" "
+if (TARGET ntuples::ntuples)
+  return()
+endif()
+get_filename_component(_IMPORT_PREFIX \"\${CMAKE_CURRENT_LIST_DIR}/../..\" ABSOLUTE)
+add_library(ntuples INTERFACE)
+add_library(ntuples::ntuples ALIAS ntuples)
+target_include_directories(ntuples INTERFACE \"\${_IMPORT_PREFIX}/include\")
+target_compile_features(ntuples INTERFACE cxx_std_20) 

```

```suggestion
target_compile_features(ntuples INTERFACE cxx_std_17) 
```

https://github.com/RPeschke/ntuples/blob/e3b124f148678fa3cfa2780579948a62df7da23e/CMakeLists.txt#L28

---

Hi,

I am sorry but it will not compile with c++17

---

> I am sorry but it will not compile with c++17

I see, in that case upstream should be fixed so that we aren't randomly doing something different than upstream. (Doesn't mean upstream needs to change before we would merge this, just that we would want upstream's approval before doing something different)

I made the original comment and pushed the change without realizing that you were also the upstream maintainer 😅. You should probably fix that and remove the attempt to set `CMAKE_CXX_STANDARD`.